src/vcpkg/binarycaching.cpp

```diff
@@ -1537,7 +1625,9 @@ namespace
             {
                 if (segments.size() != 1)
                 {
-                    return add_error("unexpected arguments: binary config 'clear' does not take arguments",
+                    return add_error(std::move(msg::format(msgUnExpectedArguments)
+                                                   .append(msg::format(msgBinaryConfig, msg::config = "clear"))
+                                                   .append(msgRequiresNoneArguments)),
```

Comment:
```suggestion
                    return add_error(msg::format(msgUnExpectedArguments)
                                                   .append(msg::format(msgBinaryConfig, msg::config = "clear"))
                                                   .append(msgRequiresNoneArguments),
```
I believe this and the several moves below of the same form are not necessary